addons/web/static/src/core/autocomplete/autocomplete.xml

```diff
@@ -17,7 +17,7 @@
                 t-on-keydown="onInputKeydown"
                 t-ref="input"
             />
-            <t t-if="isOpened">
+            <t t-if="isOpened &amp;&amp; hasOptions">
```

Comment:
you can use the word `and` => more info point 3 here https://github.com/odoo/owl/blob/master/doc/reference/templates.md#expression-evaluation